### PR TITLE
doc: bootstrap fresh environment development

### PR DESCRIPTION
**Pull request by yesnault**

### CODE REVIEW

File: docs/content/contribute/development.md [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+
+
+## NodeJS
+
+Download the latest stable Node from https://nodejs.org/en/download/current/, version >= 10.8.0
+
+Example with version 10.8.0 on OSX:
+
+```bash
+$ wget https://nodejs.org/dist/v10.8.0/node-v10.8.0-darwin-x64.tar.gz
```
Comment (bnjjj) [inline docs/content/contribute/development.md]: wget is not on OSX

Comment (bnjjj) [reply docs/content/contribute/development.md]: prefer curl

File: docs/content/contribute/development.md [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+
+Download the latest Golang version from https://golang.org/dl/
+
+Example with version 1.10.3 on OSX:
+
+```bash
+$ export GOROOT=~/go
+$ export PATH=$PATH:$GOROOT/bin
+$ cd ~
+$ wget https://dl.google.com/go/go1.10.3.darwin-amd64.tar.gz
```
Comment (bnjjj) [inline docs/content/contribute/development.md]: same, prefer curl

File: docs/content/contribute/development.md [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+```bash
+$ cd $(go env GOPATH)/src/github.com/ovh/cds/ui
+$ npm install
+```
+
+Register first user with cdsctl:
+
+```bash
+$ cdsctl signup -H http://localhost:8081 --email your-username@localhost.local --fullname yourFullname --username your-username
+# Check CDS API logs to get the validation code
```
Comment (bnjjj) [inline docs/content/contribute/development.md]: s/validation code/validation URL

Comment (bnjjj) [reply docs/content/contribute/development.md]: to fetch the password via UI

File: docs/content/contribute/development.md [hunk 4]
```diff
@@ -0,0 +1,191 @@
...
+# launch local hatchery only
+$ engine --config ~/.cds/dev.toml start hatchery:local
+```
+
+If you want to launch vcs & hooks uServices, you have to :
+
+- set name in sections `[vcs]` and `[hooks]`
+- uncomment API URL in sections `[vcs.api.http]` and `[hooks.api.http]`
+
+Of course, you have to do the same thing with uServices `repositories`, `elasticsearch`, `hatchery.swarm`, etc...
```
Comment (bnjjj) [inline docs/content/contribute/development.md]: with other uServices

File: docs/content/contribute/documentation.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
++++
+
+Documentation https://ovh.github.io/cds is generated with Hugo. Source are under https://github.com/ovh/cds/tree/master/docs/content
+
+Write / Generate / Test documentation:
+
+* Download release Hugo v32.4 https://github.com/gohugoio/hugo/releases/tag/v0.32.4 - put hugo binary in your PATH
+* Download CDS Binaries: cdsctl, engine, worker from https://github.com/ovh/cds/releases/latest
+* Clone CDS repository: git clone https://github.com/ovh/cds.git inside ${CDS_SOURCES}
```
Comment (bnjjj) [inline docs/content/contribute/documentation.md]: put command inside `

File: docs/content/contribute/development.md (hunk 3 above)
Comment (yesnault) [reply docs/content/contribute/development.md]: nope, it's a code with cdsctl

File: docs/content/contribute/development.md [hunk 6]
```diff
@@ -0,0 +1,191 @@
...
+### --> set name to "hatchery-local"
+### --> uncomment url, should be set to url = "http://localhost:8086" 
+
+```
+
+Prepare database:
+
+```bash
+$ cd $(go env GOPATH)/src/github.com/ovh/cds/engine
+$ engine database upgrade --db-password cds --db-sslmode disable
```
Comment (rayene) [inline docs/content/contribute/development.md]: I would like to understand what is the purpose of this command before typing it.

**Pull request merged**